src/update/process.rs

```diff
@@ -34,14 +36,12 @@ pub fn process_update(args: UpdateArgs) -> Result<()> {
     let client = setup_client(&sugar_config)?;
     let config_data = get_config_data(&args.config)?;
 
-    // the candy machine id specified takes precedence over the one from the cache
+    let mut cache = load_cache(&args.cache, false)?;
```

Comment:
Moving this here (from the match below) prevents running the update without the cache file, which was possible before. I think all commands after the deploy can be run without the cache file by specifying the `--candy-machine-id`.